api/src/apps/ovfiets/importer.py

```diff
@@ -0,0 +1,145 @@
+        'locationCode': 'location_code',
+        'stationCode': 'station_code',
+        'fetchTime': 'fetch_time',
+        'rentalBikes': 'rental_bikes'
+    }
+
+    def clean_key(self, key):
+        '''
+        Checks if key needs to be altered
+        '''
+        if key in self.altered_keys:
+            key = self.altered_keys[key]
+        return key
+
+    def set_to_model_field(self, model_instance, key, value):
+        '''
+        Checks if field exists then saves it to the model instance
+
+        Specific to this datasource, we dump all non existing fields to
+        the jsonfield `unmapped`
+        '''
+        try:
+            field = model_instance._meta.get_field(key)
+            setattr(model_instance, field.name, value)
+        except FieldDoesNotExist:
+            log.debug(f"Field {key} does not exist. Adding to unmapped")
+            model_instance.unmapped.update({key: value})
+
+    def clean_value(self, key, value):
+        '''
+        Parses the fetch_time timestamp and makes aware (timezone)
+        '''
+        if key == 'fetch_time':
+            value = timezone.datetime.fromtimestamp(value, tz=timezone.utc)
+        return key, value
+
+    def extract_extra(self, data_instance):
+        '''
+        Extract the nested pairs since it contains
+        the bikeRentals and fetch_time which we save
+        in the model.
+        '''
+        extra = data_instance.pop('extra')
+        data_instance.update(extra)
+        return data_instance
+
+    def extract_coordinates(self, data_instance):
+        '''
+        Extract the coordinates values and create
+        a Point object
+        '''
+        data_instance.update({
+            'geometrie': Point(
+                data_instance.pop('lng'),
+                data_instance.pop('lat')
+            )
+        })
+        return data_instance
+
+    def build_model_instance(self, data_instance, timestamp):
+        '''
+        Build a model instance with the data instance items
+        after cleaning the items.
+        '''
+        model_instance = OvFiets(scraped_at=timestamp, unmapped={})
+        data_instance = self.extract_extra(data_instance)
+        data_instance = self.extract_coordinates(data_instance)
+
+        for key, value in data_instance.items():
+            key = self.clean_key(key)
+            key, value = self.clean_value(key, value)
+            self.set_to_model_field(model_instance, key, value)
+
+        model_instance.stadsdeel = District.objects.get_district(model_instance.geometrie)
+
+        # If no district then it is not in Amsterdam. Skipping the neighbourhood saves computing time.
+        if model_instance.stadsdeel:
+            model_instance.buurt_code = Neighbourhood.objects.get_neighbourhood(model_instance.geometrie)
+
+        return model_instance
+
+    def queryset_to_model_instance_list(self, queryset):
+        '''
+        Creates a model instance list from the queryset.
```

Comment:
You need to improve the comments, preferably here and in the class comment. Specifically you should describe that you are unpacking the OVFietsRaw records, where each OVFietsRaw instance contains multiple data entries (instead of a single).